src/chain_spec.rs

```diff
@@ -452,6 +458,7 @@ pub mod general {
                 signers: bridge_signers(),
                 timelock: 10,
                 bridge_limit: (100_000_000 * POLY, 1000),
+                ..Default::default()
```

Comment:
Let's also refactor the initialization of the balances module. Instead of manually setting balances in the balances module, we should be using this new bridge genesis stuff.

Reply:
I created a separate task for this.